.gitignore

```diff
@@ -1,3 +1,6 @@
+package-lock.json
```

Comment:
We have to track these files to make sure that all the JS files are the same everywhere.